tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -369,3 +369,10 @@ func randBytes(size int) []byte {
 	}
 	return data
 }
+
+// nopStresser implements Stresser that does nothing
+type nopStresser struct{}
+
+func (s *nopStresser) Stress() error      { return nil }
+func (s *nopStresser) Cancel()            {}
+func (s *nopStresser) Report() (int, int) { return 0, 0 }
```

Comment:
`return time.Since(start).Seconds() * qps, 0`? Otherwise `failureUntilSnapshot` will error out since it expects the success rate to be increasing.